src/firewheel/control/ansible_playbooks/git.yml

```
@@ -0,0 +1,41 @@
+---
+# This playbook will clone a Git repository and move specified cached files
+# If you are using an access token, you can specify it in the `git_server` URL
+# For example: https://<token>@github.com/user/repo.git
+# The user will need to provide all relevant information including:
+# - ``git_server`` - The URL of the git server
+# - ``git_repo_path`` - The path to the repo from the server
+# - ``git_branch`` - The branch name (optional); default: main
+
+- name: Clone Git repository and move files
+  hosts: localhost
+  vars_prompt:
+    - name: git_server
+      prompt: "Please enter the Git server URL or set `ansible.git_server` in the FIREWHEEL configuration."
+      private: false
+
+    - name: git_repo_path
+      prompt: "Please enter the path to the git repository or set `ansible.git_repo_path` in the FIREWHEEL configuration."
+      private: false
+
+  tasks:
+    - name: Clone the Git repository
+      ansible.builtin.git:
+        repo: "{{ git_server }}/{{ git_repo_path }}"
+        dest: "/tmp/fwcache"
+        version: "{{ git_branch | default('main') }}"
```

Looks like doing it this way will make it default to whatever the default branch is. So, if the default branch is still master, it will pull that. If git_branch is specified, it will use that
```suggestion
        version: "{{ git_branch | default(omit) }}"
```